lib/src/api/engine/remote/ws/native.rs

```diff
@@ -110,14 +113,14 @@ impl Connection for Client {
 			#[cfg(not(any(feature = "native-tls", feature = "rustls")))]
 			let maybe_connector = None;
 
+			#[allow(deprecated)]
 			let config = WebSocketConfig {
-				max_send_queue: match capacity {
-					0 => None,
-					capacity => Some(capacity),
-				},
 				max_message_size: Some(MAX_MESSAGE_SIZE),
 				max_frame_size: Some(MAX_FRAME_SIZE),
 				accept_unmasked_frames: false,
+				write_buffer_size: Default::default(),
+				max_write_buffer_size: MAX_WRITE_BUFFER_SIZE,
+				max_send_queue: None,
```

Comment:
Could this also be reverted please 🙏

Reply:
Could you please explain why you would like this code reverted? This is the code that protects from the vulnerability that this PR aims to remove. Reverting this code would defeat the purpose of the PR.

Reply:
Is it just the Default::default)() part?

Reply:
It was about max_send_queue, but already explained thank you

Reply:
Thanks, sorry!